ibm/service/atracker/data_source_ibm_atracker_targets_test.go

```diff
@@ -94,10 +95,20 @@ func testAccCheckIBMAtrackerTargetsDataSourceConfig(targetName string, targetTar
 				api_key = "xxxxxxxxxxxxxx"
 				service_to_service_enabled = true
 			}
+			logdna_endpoint {
+				target_crn = "crn:v1:bluemix:public:logdna:us-south:a/11111111111111111111111111111111:22222222-2222-2222-2222-222222222222::"
+				ingestion_key = "xxxxxxxxxxxxxx"
```

Comment:
Instead of marking "xxxxxx" take the values from varaibles defined in provider_test.go file.
While you run the testcases set those values and run them

Reply:
sorry the name if the file is "acctest.go" under acctest folder